### PR TITLE
Verify Respondent Details Automation

### PR DESCRIPTION
**Pull request by SharifChoudhury1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/end-to-end/pages/amendRespondentDetails/amendRespondentDetails.js [hunk 1]
```diff
@@ -5,15 +5,14 @@ const testConfig = require("../../../config");
...
+    I.fillField('#responseReceivedDate-year', commonConfig.caseAcceptedYear);
+    I.fillField('#respondentCollection_0_responseRespondentAddress_responseRespondentAddress_postcodeInput', commonConfig.respondentPostCode);
+    I.click('#respondentCollection_0_responseRespondentAddress_responseRespondentAddress_postcodeLookup > button');
+    I.wait(2);
     I.waitForText(commonConfig.respondentAddress, testConfig.TestTimeToWaitForText);
-    await I.selectOption('#respondentCollection_0_responseRespondentAddress_responseRespondentAddress_addressList', commonConfig.respondentAddress);
-    await I.click(commonConfig.continue);
-    await I.click(commonConfig.submit)
+    I.selectOption('#respondentCollection_0_responseRespondentAddress_responseRespondentAddress_addressList', commonConfig.respondentAddress);
+    I.click(commonConfig.continue);
```
Comment (pats-john) [inline src/test/end-to-end/pages/amendRespondentDetails/amendRespondentDetails.js:16]: No Check for the Filled Data of the labels of this page...

File: src/test/end-to-end/pages/amendRespondentDetails/amendRespondentDetails.js [hunk 2]
```diff
@@ -5,15 +5,14 @@ const testConfig = require("../../../config");
 module.exports = async function () {
 
     const I = this;
-    await I.fillField('#responseReceivedDate-day', commonConfig.caseAcceptedDay);
```
Comment (pats-john) [inline src/test/end-to-end/pages/amendRespondentDetails/amendRespondentDetails.js:8]: Please check the labels and text for this page...

**Review by pats-john — CHANGES_REQUESTED** [review]

File: src/test/end-to-end/paths/verifyRespondentDetails.js [hunk 3]
```diff
@@ -12,5 +12,4 @@ Scenario('Verify Respondent Details', async ({I}) => {
 
     await claimantRespondentDetails(I, eventNames.CLAIMANT_RESPONDENT_DETAILS);
```
Comment (pats-john) [inline src/test/end-to-end/paths/verifyRespondentDetails.js:13]: Remove unwanted Awaits...

**Review by pats-john — APPROVED** [review]

**Review by pats-john — APPROVED** [review]

**Review by dharrisMOJ — APPROVED** [review]

File: src/test/end-to-end/paths/verifyRespondentDetails.js [hunk 4]
```diff
@@ -12,5 +12,4 @@ Scenario('Verify Respondent Details', async ({I}) => {
...
 
-}).tag('@biggerrefactoring');
-    //.retry(testConfig.TestRetryScenarios);
```
Comment (pats-john) [inline src/test/end-to-end/paths/verifyRespondentDetails.js:16]: Can you please tag this with RET-BAT as well...

**Review by pats-john — CHANGES_REQUESTED** [review]

File: src/test/end-to-end/paths/verifyRespondentDetails.js (hunk 4 above)
Comment (SharifChoudhury1) [reply src/test/end-to-end/paths/verifyRespondentDetails.js:16]: RET-BAT was tagged

**Review by SunnyAyeni — APPROVED** [review]

**Review by SharifChoudhury1 — COMMENTED** [review]
done

**Review by pats-john — APPROVED** [review]

**Pull request merged**